website/preprints/model.py

```diff
@@ -178,16 +178,17 @@ def set_published(self, published, auth, save=False):
 
     def set_preprint_license(self, license_detail, auth, save=False):
 
-        set_license(self, license_detail, auth, node_type='preprint')
-
-        self.node.add_log(
-            action=NodeLog.PREPRINT_LICENSE_UPDATED,
-            params={
-                'preprint': self._id
-            },
-            auth=auth,
-            save=False
-        )
+        license_record, license_changed = set_license(self, license_detail, auth, node_type='preprint')
+
+        if license_changed:
+            self.node.add_log(
+                action=NodeLog.PREPRINT_LICENSE_UPDATED,
+                params={
+                    'preprint': self._id
+                },
```

Comment:
Discuss: Would it make sense to store the `license_record` here? It seems like the sort of thing we should be logging -- `"<User> updated <preprint> license to <license>"` instead of just `"<User> updated <preprint> license"`

It wouldn't be necessary to display it yet; until `NodeLogParamsSerializer` and `logActionsList.json` are updated it won't be apparent that it's being stored.

See if @saradbowman or anyone else from Product has opinions about how this log should be displayed.